CMSgov/onemac
# Oy2 9209 cms rev reg

Story: https://qmacbis.atlassian.net/browse/OY2-9209
Endpoint: https://d2zzhqlsns6nm.cloudfront.net

### Details

Added CMS Reviewer Option to CMS users and changed State option styles to match new figma design 

### Test Plan

1. Login as cmsapproverunregistered and make sure you are given both options for the CMS user (CMS Approver & CMS Reviewer)
2. Click on CMS Reviewer and make sure it goes to signup/cmsreviewer (should be a blank page unless George's PR gets merged in)
3. Login as stateuserunregistered and make sure the options match the new style
4. Make sure the page's back button works and goes to pages you were previously at
